src/gke/run_batch.sh

```diff
@@ -65,6 +65,20 @@ fi
 if [[ -z $INFERENCING_IMAGE_URI ]]; then
   export INFERENCING_IMAGE_URI=$INFERENCE_IMAGE
 fi
+if [[ -z $GKE_VERSION ]]; then
+  export GKE_VERSION="1.26.3-gke.1000"
+fi
+
+TERRAFORM_DIRECTORY_NAME="aiinfra-terraform-$PROJECT_ID"
+GS_REGEX_MATCH='^gs://([a-z0-9_\.-]+)/.*'
+if [[ $CONVERTED_MODEL_UPLOAD_PATH =~ GS_REGEX_MATCH ]]; then
+  TARGET_BUCKET_NAME="${BASH_REMATCH[1]}"
+elif [[ $MODEL_SOURCE_PATH =~ $GS_REGEX_MATCH ]]; then
+  TARGET_BUCKET_NAME="${BASH_REMATCH[1]}"
+else
+  TARGET_BUCKET_NAME=$TERRAFORM_DIRECTORY_NAME
+fi
+export TERRAFORM_GCS_PATH="gs://$TARGET_BUCKET_NAME/$TERRAFORM_DIRECTORY_NAME"
```

Comment:
Let's make the behavior simpler by requiring the user to explicitly set TERRAFORM_GCS_PATH (preferably passed as a parameter). Using the bucket from model path might not be expected by the user.

Reply:
This functionality is already available if the user just manually sets the TERRAFORM_GCS_PATH env var since this is built off of the CPT image. I'll call it out in the readme but I don't think we should make this a required param.